core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -4424,6 +4424,7 @@ private int prepareLatestByColumnIndexes(ObjList<ExpressionNode> latestBy, Recor
                     case ColumnType.SHORT:
                     case ColumnType.INT:
                     case ColumnType.LONG:
+                    case ColumnType.TIMESTAMP:
```

Comment:
nit: if we support TIMESTAMP, we should do the same for DATE as both are `long` internally.

Reply:
I can do that, yes.

Reply:
I extended the change to BYTE, DATE, FLOAT, DOUBLE & GEOHASH .

Reply:
Resolved thanks to @bziobrowski